test/controllers/api/messages_controller_test.rb

```diff
@@ -0,0 +1,570 @@
+require "test_helper"
+
+module Api
+  class MessagesControllerTest < ActionDispatch::IntegrationTest
+    def setup
```

Comment:
Has this been copied from notes and not removed?

Reply:
This still seems to be present? What was the resolution?

Reply:
Sorry, I guess there were 2 mentions of notes, and I only removed one.